UPGRADING.md

```diff
@@ -0,0 +1,105 @@
+# Upgrading
+
+This section describes how to upgrade a Conjur Server.
+
+## Standard Process
+
+The following steps describe a standard upgrade of the Conjur server, when deployed
+using Docker Compose. These steps assume you have defined your Conjur image in
+a service named `conjur`, and that you have access to the Conjur data key
+that was used when you originally deployed your Conjur server.
+1. In your terminal window, set the `CONJUR_DATA_KEY` environment variable:
+   ```
+   export CONJUR_DATA_KEY={your Conjur data key}
+   ```
+
+1. Edit the Conjur image version in `docker-compose.yml` to reference the new
+   version.
+
+1. Pull the new Conjur image version:
+   ```
+   docker-compose pull conjur
+   ```
+
+1. Stop the Conjur container:
+   ```
+   docker-compose stop conjur
+   ```
+
+1. Bring up the Conjur service using the new image version without changing
+   linked services:
+   ```
+   docker-compose up -d --no-deps conjur
+   ```
+
+1. View Docker containers and verify all are healthy, up and running:
+   ```
+   docker ps -a
```

Comment:
Suggest using `docker-compose` as we assum user does
`docker-compose ps -a`
![image](https://user-images.githubusercontent.com/50704381/87643836-08615c00-c754-11ea-8f29-7840ef08853f.png)